docs/development/install.rst

```diff
@@ -184,3 +184,37 @@ For building this documentation,
 see :doc:`docs`.
 
 And for setting up for the front end development, see :doc:`standards`.
+
+Troubleshooting
+---------------
+
+You might face issues while installing.
+
+Error: `SocialApp matching query does not exist.`
+
+Please follow the guidelines for installation given here:
+
+- Start the installation
+  Run the installation: `sudo python3 manage.py runserver 0.0.0.0:<number>`
+
+- Visit the readthedocs DJANGO Admin site:
+  `http://localhost:<number>/admin`
+
+- EDIT the SITES section:
```

Comment:
Not sure we actually need to do anything with the `sites`

Reply:
Could you run the setup and see if you get any errors?